src/util/PathUtil.h

```diff
@@ -106,6 +106,7 @@ void openFileWithFilebrowser(const fs::path& filename);
 
 [[maybe_unused]] fs::path ensureFolderExists(const fs::path& p);
 
+fs::path getLongPath(const fs::path& path);
```

Comment:
A short doc comment like
```c++
/**
 *  Expand symbols in given path for platform compatibility. Call this before
 * passing a path to another program.
 */
```
would be helpful here!

Other than that, this PR looks good to me!

Reply:
I don't know if it is important here, but I think that `expand` referrs to different operation ([ExpandEnvironmentStringsW](https://docs.microsoft.com/en-us/windows/win32/api/processenv/nf-processenv-expandenvironmentstringsw)). Maybe it would be better to just word it like `Convert to platform compatible path`?

Reply:
Thar's better. I agree.